scanner/decisions/0014-csaf-enricher.md

```diff
@@ -0,0 +1,88 @@
+
+The enricher will fetch Red Hat advisories and extract data we have determined has been inconsistent:
+
+* Description
+  * The current implementation takes the description from the CVE, so if a package is affected by two different CVEs
+    associated with the same advisory, then there is a clear inconsistency.
+* Severity
+  * The current implementation takes the severity from the CVE, so if a package is affected by two different CVEs
+    rated with different severities, then there is a clear inconsistency.
+  * Claircore does not provide us a way to read the severity at .document.aggregate_severity.text at this time,
+    so we must calculate it. Luckily, it is pretty clear how to do so: Red Hat advisories are given the highest severity rating
+    based on the associated CVEs.
+  * Note: it is very possible a CVE has two different severity ratings, depending on the product.
+    For example: https://access.redhat.com/security/cve/CVE-2023-3899 is rated Important, in general,
+    but Moderate for subscription-manager in RHEL 7. For this case, the OVAL v2 entry in 
+    https://security.access.redhat.com/data/oval/v2/RHEL7/rhel-7-including-unpatched.oval.xml.bz2
+    for the associated RHSA, RHSA-2023:4701, actually has the expected, correct rating of Moderate.
+    The CSAF entry in https://security.access.redhat.com/data/csaf/v2/advisories/2023/rhsa-2023_4701.json
+    also lists the Moderate severity rating under CVE-2023-3899's "threats" entry.
+* CVSS vectors and scores
+  * Same idea as Severity.
+  * Red Hat advisories are not given a CVSS score, so we calculate it as done prior to 4.6.0:
+    * Pick the highest CVSS scores (v3 and v2) from the associated CVEs.
+    * Note: it is very possible a CVE has two different CVSS scores, depending on the product.
+      For example: https://access.redhat.com/security/cve/CVE-2023-3899 is scored 7.8, in general,
+      but 6.1 for subscription-manager in RHEL 7. For this case, the OVAL v2 entry in 
+      https://security.access.redhat.com/data/oval/v2/RHEL7/rhel-7-including-unpatched.oval.xml.bz2
+      for the associated RHSA, RHSA-2023:4701, actually has the general CVSS score (7.8) instead of the true score (6.1).
+      Meanwhile, the CSAF entry in https://security.access.redhat.com/data/csaf/v2/advisories/2023/rhsa-2023_4701.json
+      lists the true, correct score of 6.1. So, the output we get here will differ from the previous OVAL v2-based output, 
+      but it will be more accurate.
+    * Note: we will not run into a case where an advisory has two different scores for the same CVE.
+      CVEs are given an overall score which may be overridden for specific product. Since advisories
+      are per-product, a single advisory cannot be associated with scores related to other products.
+
+If we discover other fields have shown inconsistencies, we will include them, as-needed.
```

Comment:
Skimming the UI - published/updated dates and links may be others (see in staging some have NVD links, others have RHSA links)